.github/workflows/ci.yml

```
@@ -0,0 +1,38 @@
+name: Python CI
```

YAMLlint (1.38.0)

[error] wrong new line character: expected \n

(new-lines)